### PR TITLE
Gitea Automation using OpenFaaS Blog post

### PR DESCRIPTION
**Pull request by techknowlogick**
## Description
A new blog post describing Gitea automation using OpenFaaS

## Motivation and Context
The Gitea project is moving to self-hosting development, and we will use the automation we currently have which is tied to Github, and so we will need to write our own. OpenFaaS is a vendor-neutral way for us to write automation to quickly write those apps. I've written this blog post to share an example of an integration to inspire others.

## Have you applied the editorial and style guide to your post?

Yes.

## How have you tested the instructions for any tutorial steps?

Copy/pasting the instructions from blog post and running things as stated

## Types of changes

- [x] New blog post

## Checklist:

- [x] I have given attribution for any images I have used and have permission to use them under Copyright law
- [x] My code follows the [writing-style of the publication](README.md) and I have checked this
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: _posts/2021-01-31-gitea-faas.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
+---
+title: "Gitea automation using OpenFaaS"
+description: "Switching to Gitea doesn’t mean having to give up your bot automations"
+date: 2021-01-31
```
Comment (techknowlogick) [inline _posts/2021-01-31-gitea-faas.md:4]: I picked the date `2021-01-31` arbitrarily.

**Comment by alexellis** [comment]
This PR makes me very happy.

> Gitea automation using OpenFaaS

Do you have another suggestion for the title that is more use-case focused, or snappier? 

Extend and automate self-hosted Gitea with functions
How to write bots for Gitea with OpenFaaS

These kinds of websites can look trashy, but there is some gold in here -> https://www.dreamgrow.com/seven-easy-tricks-to-write-catchy-headlines/

>2. Be Clear About Your Main Benefit
> 4. Questions in the Headline
>5. Appeal to You Reader’s Hunger for Knowledge
>6. Now You Can Have [Good Thing] and [Other Good Thing]


And do you mind covering faasd briefly as an alternative deployment option? Use-case - you have Gitea somewhere on your public or private network, and don't want to learn K8s just to add some webhook automations.

**Comment by techknowlogick** [comment]
Thanks for the tips :)

I've just switched to using faasd as an example, and updated the title.

**Review by alexellis — APPROVED** [review]
LGTM

**Pull request merged**